notifications-sample/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log
+
+All notable changes to the "notifications-sample" extension will be documented in this file.
+
+Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+
+## [Unreleased]
+
+- Initial release
```

Comment:
The sample guidelines don't say anything about keeping or removing the CHANGELOG file, but most of the sample extensions in this repo don't have one so I'd probably recommend removing it as well since it doesn't really add anything meaningful to this sample.